src/screens/ConstructionSiteDetailScreen.js

```diff
@@ -58,16 +60,28 @@ export const ConstructionSiteDetailScreen = ({ route }) => {
     restrictions,
     startDate,
     title
-  } = constructionSites[index];
-
-  useMatomoTrackScreenView(`${MATOMO_TRACKING.SCREEN_VIEW.CONSTRUCTION_SITE_DETAIL} / ${title}`);
+  } = constructionSites[0];
 
   const extendedTitle = (category ? `${category} - ` : '') + title;
   const formattedDates = formatDates(startDate, endDate);
 
+  if (loading && !refreshing) {
+    return <LoadingSpinner loading />;
+  }
+
   return (
     <SafeAreaViewFlex>
-      <ScrollView keyboardShouldPersistTaps="handled">
+      <ScrollView
+        keyboardShouldPersistTaps="handled"
```

Comment:
we do not have the need for some keyboard stuff on this screen, i guess.

Reply:
removed it 🪓